xml/http_servers.xml

```diff
@@ -2623,6 +2623,17 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:encode:uvicorn:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^WSGIServer/((?:\d+\.)*\d+)\s*C?Python/((?:\d+\.)*\d+)$">
```

Comment:
Matches `WSGIServer/0.2                          CPython/3.10.5` but I think that's fine.

Reply:
It should match `WSGIServer/0.2 CPython/3.10.5` as that's one of the examples. :)

Reply:
Oops, I copied the wrong example. I think it was a similar multiple dot situation.